tutorials/data-science-exploration/index.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Exploratory queries with Google BigQuery
+description: Use BigQuery to investigate arbitrarily large datasets.
+author: jerjou
+tags: Data Science, BigQuery
+date_published: 2017-05-10
+---
+[Google BigQuery](/bigquery) is a fully-managed service for data
+exploration using an SQL syntax. Once data has been loaded into the service, it
+can be queried at interactive speeds, without having to know beforehand what
+you're looking for.
+
+There has been much already written about using BigQuery for data analysis - for
+example:
+
+* [Analyzing Financial Time Series using BigQuery](/solutions/time-series/bigquery-financial-forex)
+* [How to forecast demand with Google BigQuery, public datasets and TensorFlow](/blog/big-data/2016/05/how-to-forecast-demand-with-google-bigquery-public-datasets-and-tensorflow)
+* Any of the articles around the [BigQuery Public Datasets](/bigquery/public-data/)
```

Comment:
When I follow the link, it isn't immediately obvious to me what I'm supposed to be looking at from the perspective of this article. Can you be more explicit, or provide a canonical example of the articles that you're talking about?